src/js/notifier.js

```diff
@@ -154,7 +154,15 @@
                         reflow = this.element.offsetWidth;
                         addClass(this.element, classes.visible);
                         // attach click event
-                        on(this.element, 'click', this.__internal.clickHandler);
+                        if (!alertify.defaults.notifier.closeButton) {
+                            on(this.element, 'click', this.__internal.clickHandler);
+                        } else {
+                            var close = document.createElement('span');
+                            addClass(close, 'ajs-close');
```

Comment:
Update [classes](https://github.com/MohammadYounes/AlertifyJS/blob/master/src/js/dialog/intro.js#L30-L55) variable to include the new classes, rather than using string literals.